docs/source/contents/kubernetes/storage-secrets/index.md

```diff
@@ -0,0 +1,147 @@
+Some storage types have multiple providers, such as `s3` having AWS S3 itself, MinIO, Ceph, and so on.
+
+The remote **name** is your choice.
+The prefix you use for models in `spec.storageUri` must be the same as this remote name.
+
+The remote type is one of the values [supported by Rclone](https://rclone.org/docs/).
+For example, for AWS S3 it is `s3` and for Dropbox it is `dropbox`.
+
+The provider parameters depend entirely on the remote _type_ and the specific _provider_ you are using.
+Please check the Rclone documentation for the appropriate provider.
+Note that Rclone docs for storage types call the parameters _properties_ and provide both _config_ and _env var_ formats--you need to use the _config_ format.
+For example, the GCS parameter `--gcs-client-id` described [here](https://rclone.org/googlecloudstorage/#gcs-client-id) should be used as `client_id`.
+
+For reference, this format is described in the [Rclone documentation](https://rclone.org/rc/#config-create).
+Note that we do not support the use of `opts` discussed in that section.
+
+## Kubernetes Secrets
+
+Kubernetes Secrets are used to store Rclone configurations, or _storage secrets_, for use by Servers.
+Each Secret should contain **exactly one** Rclone configuration.
+
+A Server can use storage secrets in one of two ways:
+* It can dynamically load a secret specified by a Model in its `.spec.secretName`
+* It can use global configurations made available via [preloaded secrets](#preloaded-secrets)
+
+The name of a Secret is entirely your choice, as is the name of the data key in that Secret.
+All that matters is that there is a single data key and that its value is in the format described above.
+
+```{note}
+It is possible to use preloaded secrets for some Models and dynamically loaded secrets for others.
+```
+
+### Preloaded Secrets
+
+Rather than Models always having to specify which secret to use, a Server can load storage secrets ahead of time.
+These can then be reused across many Models.
+
+When using a preloaded secret, the Model definition should leave `.spec.secretName` empty.
+The protocol prefix in `.spec.storageUri` still needs to match the remote name specified by a storage secret.
+
+The secrets to preload are named in a centralised ConfigMap called `seldon-agent`.
+This ConfigMap applies to _all_ Servers managed by the same SeldonRuntime.
+By default this ConfigMap only includes `seldon-rclone-gs-public`, but can be extended with your own secrets as shown below:
+
+```{literalinclude} ../../../../../samples/auth/agent.yaml
+:language: yaml
+```
+
+The easiest way to change this is to update your SeldonRuntime.
+* If you are using Helm, the corresponding value is `config.agentConfig.rclone.configSecrets`:
+  ```{literalinclude} ../../../../../k8s/helm-charts/seldon-core-v2-runtime/values.yaml
+  :language: yaml
+  :start-at: config
+  :end-at: configSecrets
+  ```
+* Otherwise, you can add secrets to `.spec.config.agentConfig.rclone.config_secrets`:
```

Comment:
```suggestion
* Otherwise, if you directly manage `SeldonRuntime` resource you can add secrets to its definition at `.spec.config.agentConfig.rclone.config_secrets`:
```

Reply:
Fixed in https://github.com/SeldonIO/seldon-core/pull/4969/commits/cadf481d2afab78d6982cc92c5ac32196ad309fe